.github/workflows/deploy.yml

```diff
@@ -124,7 +128,7 @@ jobs:
         --outdir dist/
         
     - name: Publish package to PyPI
-      uses: pypa/gh-action-pypi-publish@master      
+      uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
why this change? Does it work?

Reply:
Because I don't want to run off of their master branch. I'd rather depend on specific, released versions.